### ISSUE TITLE
check for codes before writing header

### ISSUE THREAD

**Issue by terrydavis903** [useflyent/fhttp#10]
in server.go, check if code < 100 or > 999, before checkWriteHeaderCode(code)

there are times it passes code 0, which will hangup socket on client end, and crash server end

**Comment by zMrKrabz** [comment]
I don't believe that is an error with fhttp, think that is in the scope of net/http?
